content/en/tracing/profiler/enabling/ddprof.md

```diff
@@ -14,34 +16,45 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-The Datadog Profiler for Linux is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+The compiled languages profiler, <code>ddprof</code> is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
 </div>
 
-The Datadog Profiler for Linux lets you collect profile data for applications running on Linux. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust. It can also profile services such as nginx, Redis, or Postgres, as it does not require access to source code.
+The Datadog Profiler for compiled languages (`ddprof`) uses OS level APIs to collect profiling data. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust.
```

Comment:
What's the product name here?
* Datadog Profiler
* Datadog Profiler for Compiled Languages

I'd guess the latter and suggest that the casing should follow, but I actually have no idea!  I think this is the first Profiling product where the target (runtime?) wasn't already a proper name!

Reply:
Ah, the name. I believe the official name of the entire feature/product is (Datadog) Continuous Profiler. And the specific language libraries each contain a profiler _for lang_ feature that you can enable. which we can also refer to as the _lang_ profiler (or sometimes _lang_ Profiler). This particular profiler is not alone in being all over the map in terms of capitalization and naming used in the docs. So, throughout these docs I have tried to keep the nouns common (uncapitalized) except in H1/navigation where we Title Case things. So, I would call these the docs for the "profiler for compiled languages" (no caps). I'll do a spot check for consistency here, but getting consistency across the whole docs is a bigger project :)

Reply:
Gotcha!  Thanks for the clarification!